httpdocs/ajax/proto/refresh_protos.php

```diff
@@ -0,0 +1,20 @@
+<?php # This script refresh all protos
+
+require '../../../php/refresh_password.php';
+if ($_GET['password'] === $refresh_password) {
+  require '../../../php/github_oauth.php';
+  require '../../../php/github_asset.php';
+  require '../../../php/database.php';
+  require '../../../php/create_or_update.php';
+
+  $mysqli = new mysqli($database_host, $database_username, $database_password, $database_name);
+  if ($mysqli->connect_errno)
+    error("Can't connect to MySQL database: $mysqli->connect_error");
+  $mysqli->set_charset('utf8');
+  $result = $mysqli->query('SELECT id, url FROM proto where branch="proto"') or error($mysqli->error);
```

Comment:
```suggestion
  $result = $mysqli->query('SELECT id, url FROM proto ') or error($mysqli->error);
```

Once reviewed and approved, we should change it such that the protos of every branches are regularly updated.
Note: as we will more than double the number of refreshed protos (from ~500 to ~1200), it is possible that we encounter some time out issue.